site/en/workflows.md

```diff
@@ -1407,7 +1581,7 @@ If your computer is not that powerful you may try this workflow. It works fine f
     <tr>
       <td>14</td>
```

Comment:
Renumber?

Reply:
The idea is to reference the step number from the guide before the recommendation. I'll make internal links of ### ### the numbers to make that clearer.